.github/renovate.json

```diff
@@ -1,13 +1,36 @@
 {
-  "extends": ["local>brave/renovate-config"],
-  "labels": ["dependencies", "renovate"],
+  "extends": [
+    "local>brave/renovate-config"
+  ],
+  "labels": [
+    "dependencies",
+    "renovate"
+  ],
   "prConcurrentLimit": 4,
   "rangeStrategy": "pin",
   "packageRules": [
     {
-      "matchDepTypes": ["peerDependencies"],
+      "matchDepTypes": [
+        "peerDependencies"
+      ],
       "enabled": false
+    },
+    {
+      "matchUpdateTypes": [
+        "minor",
+        "patch"
```

Comment:
I'm not opposed to adding `"digest"` and `"pin"` to this list, but I'm a bit hazy on exactly what they mean.

The build will fail if something goes wrong here.